### ISSUE TITLE
Warning on the top

### ISSUE THREAD

**Issue by ahmadawais** [release-it/release-it#627]
Latest 13.3 version once run outputs this at the top;

```sh
Passing an array of plugins to Octokit.plugin() has been deprecated.
Instead of:
  Octokit.plugin([plugin1, plugin2, ...])
Use:
  Octokit.plugin(plugin1, plugin2, ...)

…then all the output
```

**Comment by ahmadawais** [comment]
![](https://on.ahmda.ws/46d01c/c)

**Comment by webpro** [comment]
Interesting, I don't see this here when using octokit for GitHub Releases. What's `rlz` doing and/or what is the config you're using?

**Comment by jchamb** [comment]
Same error for me off a fresh install this morning. Really basic config:

```
{
 "plugins": {
    "./scripts/release-it-variable-build.js": {
      "productionCmd": "yarn build",
      "preReleaseCmd": "yarn build-staging"
    }
  }
}
```

**Comment by webpro** [comment]
Alright, fixed in [v13.3.1](https://github.com/release-it/release-it/releases/tag/13.3.1) (https://github.com/octokit/rest.js/issues/1654)
